src/mixing_fab.h

```diff
@@ -0,0 +1,158 @@
+
+#ifndef CYCAMORE_SRC_MIXING_FAB_H_
+#define CYCAMORE_SRC_MIXING_FAB_H_
+
+#include <string>
+#include "cyclus.h"
+#include "cycamore_version.h"
+
+namespace cycamore {
+  
+  /// MixingFab takes in N streams of material and mixes them accordingly
+  /// to the ratios prodided by the user.
+  ///
+  /// The MixingFab has N input inventories: one for each streams to be mixed,
+  /// and one output stream.
+  ///
+  /// The supplying of mixed material is constrained by available inventory of
+  /// mixed material quantities.
```

Comment:
I would just make this doc comment all one paragraph.  Make sure to mention throughput constraint, output buffer being full constraint, and that the user can set input and output buffer sizes.